src/main/kotlin/me/zeroeightsix/kami/feature/module/ItemHighlight.kt

```diff
@@ -0,0 +1,21 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import me.zeroeightsix.kami.Colour
+import me.zeroeightsix.kami.target.ItemCategory
+import me.zeroeightsix.kami.target.ItemSupplier
+
+@Module.Info(
+    name = "ItemHighlight",
+    category = Module.Category.RENDER,
+    description = "Highlights certain Item Slots"
+)
+object ItemHighlight: Module() {
```

Comment:
Add a `@see MixinHandledScreen` javadoc to this module - otherwise it appears empty.

Reply:
just as a note, i had to use the full name of the class so intelliJ finds it

Reply:
You have to import it

Reply:
im talking about the javadoc here and i dont think importing is a thing there

Reply:
Imported items don't need a fully qualified names in docs, but it is true that IntelliJ won't import from javadocs - this is fine.